tests/queries/0_stateless/01259_default_expressions_file_engine.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
There is also `user_files_path` where it's allowed to read/write files.
You can create a file there also with `file` table function.